src/task_queue/nudges/contents_spacing_correction.py

```diff
@@ -0,0 +1,44 @@
+from database.models import Action, Event, Testimonial
+
+
+def correct_spacing_in_actions():
+    actions = Action.objects.filter(is_deleted=False)
+    for action in actions:
+        about = action.about.replace("<p><br></p>", "").replace("<p>.</p>", "").replace("<p>&nbsp;</p>", "")
+        deep_dive = action.deep_dive.replace("<p><br></p>", "").replace("<p>.</p>", "").replace("<p>&nbsp;</p>", "")
+        action.about = about
+        action.deep_dive = deep_dive
+        action.save()
+    return actions.count()
+
+
+
+def correct_spacing_in_events():
+    events = Event.objects.filter(is_deleted=False)
+    for event in events:
+        description = event.description.replace("<p><br></p>", "").replace("<p>.</p>", "").replace("<p>&nbsp;</p>", "")
+        event.description = description
+        event.save()
+    return events.count()
+
+
+def correct_spacing_in_testimonials():
+    testimonials = Testimonial.objects.filter(is_deleted=False)
+    for testimonial in testimonials:
+        body = testimonial.body.replace("<p><br></p>", "").replace("<p>.</p>", "").replace("<p>&nbsp;</p>", "")
+        testimonial.body = body
+        testimonial.save()
+    return testimonials.count()
+
+def correct_contents_spacing():
```

Comment:
Add a feature flag update-html-format, so we can update first one community then others.
Also we want to be able to run and get a report of what content in what communities would be updated first, before actually updating the content.  That could use a separate feature flag check-html-format

Reply:
@abdullai-t - what I meant is to have one task, which always will send the report, but if the feature flag is set, it will do the updates for that communities content.